client/styles/components/_preferences.scss

```diff
@@ -35,7 +35,8 @@
 .preference {
 	display: flex;
 	flex-wrap: wrap;
-	justify-content: space-between;
+	padding-bottom: #{40 / $base-font-size}rem;
+	border-bottom: 2px dashed $light-button-border-color;
```

Comment:
To remove the border from the last preference, all you have to do is change this line to:
```scss
& + & {
  border-top: 2px dashed $light-button-border-color;
}
```